dotcom-rendering/src/web/lib/notification.ts

```diff
@@ -52,31 +44,20 @@ export const addNotificationsToDropdownLinks = (
 	const notificationsByTarget = groupNotificationsByTarget(notifications);
 
 	const linksWithNotifications = links.map((link) => {
-		const targetHasNewNotifications = Object.prototype.hasOwnProperty.call(
-			notificationsByTarget,
-			link.id,
+		const newNotifications = (link.notifications ?? []).concat(
```

Comment:
I guess maybe just `const notifications` here would make sense as a name, since this is now _all_ then notifications for a link, not just the new ones?

Reply:
Sadly that shadows the parameter of the same name which is verboten in our ESLint rules.